xmtp_mls/src/groups/mls_sync.rs

```diff
@@ -1770,6 +1770,7 @@ where
                         installation_key,
                         data: encrypted,
                         hpke_public_key: installation.hpke_public_key,
+                        wrapper_algorithm: WelcomeWrapperAlgorithm::Curve25519.into(),
```

Comment:
Temporarily defaulting to Curve25519 in this PR, which gets set dynamically in up-stack PRs.